features/net/FEATURE_IPV4/lwip-interface/.mbedignore

```diff
@@ -0,0 +1,6 @@
+lwip/doc/*
+lwip/test/*
+lwip/src/apps/*
+lwip/src/netif/ppp/*
+lwip/src/inlude/lwip/apps/*
+lwip/src/inlude/posix/*
```

Comment:
If the lwip codebase is being mirrored, shouldn't we just avoid adding these directories into mbed-os?

None of these seem to add a benefit except maybe documentation, which shouldn't present a compilation issue.